Pear/Profile/ChatStatus.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ChatStatus.swift
+//  Pear
+//
+//  Created by Lucy Xu on 3/14/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+enum ChatStatus {
+    /// No one has reached out yet
+    case planning
+    /// No one has reached out in 3 days
+    case noResponses
+
+    /// User already reached out and is waiting on `SubUser`
+    case waitingOn(User)
```

Comment:
```suggestion
    case waiting(on: User)
```

On or for? On is a little colloquial, no?